tests/functional/spec/previous.spec.js

```diff
@@ -0,0 +1,52 @@
+const helpers = require('../helpers');
+const EmploymentStatusBlockPage = require('../generated_pages/hub_and_spoke/employment-status.page.js');
+const EmploymentTypePage = require('../generated_pages/hub_and_spoke/employment-type.page.js');
+const HubPage = require('../base_pages/hub.page.js');
+const ProxyPage = require('../generated_pages/hub_and_spoke/proxy.page.js');
+const schema = 'test_hub_complete_sections.json';
+
+describe('Choose another section link', function () {
+
+  it('When a user gets to initial question, then the previous location link should not be displayed', function () {
+    return helpers.openQuestionnaire(schema)
```

Comment:
It's worth doing a beforeEach in the describe function to open the question schema and reduce duplication

Reply:
I forgot about what you told me the other day, sorry for that. It's fixed now.